backends/dpdk/midend.cpp

```diff
@@ -89,10 +90,26 @@ class EnumOn32Bits : public P4::ChooseEnumRepresentation {
     EnumOn32Bits() {}
 };
 
+/**
+This class implements a policy suitable for the ConvertErrors pass.
+The policy is: convert all errors to bit<16>
+*/
+class ErrorBits : public P4::ChooseErrorRepresentation {
```

Comment:
I would call this ErrorOn16Bits or something similar.
Alternatively, the class should be called ErrorWidth and have a constructor argument for the width.